model/src/channel/message/kind.rs

```diff
@@ -1,25 +1,46 @@
 use crate::channel::ConversionError;
 use serde_repr::{Deserialize_repr, Serialize_repr};
 
+/// Type of a [`Message`].
+///
+/// [`Message`]: super::Message
+#[allow(missing_docs)]
```

Comment:
TODO, but I don't feel comfortable documenting the rest, especially `ThreadCreated` and `ThreadStarterMessage`

Reply:
we can look into that another time, i don't know either